app/controllers/v2/link_sets_controller.rb

```diff
@@ -12,6 +12,14 @@ def put_links
       render status: response.code, json: response
     end
 
+    def get_linked
+      render json: Queries::GetLinked.new(
+        content_id: content_id,
+        link_type: params.fetch(:link_type),
+        fields: params.fetch(:fields),
+      ).call
+    end
```

Comment:
you are right, going to do it. Can you advise where I should return the 4xx response? Here in the controller or in the query itself?

I'd say here.